src/utils/L.Map.Sync.js

```diff
@@ -37,27 +35,29 @@ L.Map.include({
                 const newPos = getPosition(this._element)
                 const changePos = oldPos.subtract(newPos)
 
-                syncedMaps
-                    .filter(map => map !== origin)
-                    .forEach(map => {
+                syncedMaps.forEach(map => {
+                    if (map === origin) {
```

Comment:
This needs to be `!==`, not `===`, unless we switch to an early return!

Reply:
Sorry, a bit quick there. Fixed!

Reply:
This probably needs to be pushed?